updater/versions/update-1.py

```diff
@@ -197,6 +210,10 @@ def write_lepidopter_update_logrotate():
     with open(LEPIDOPTER_UPDATE_LOGROTATE_PATH, "w") as out_file:
         out_file.write(LEPIDOPTER_UPDATE_LOGROTATE)
 
+def write_lepidopter_update_logrotate():
```

Comment:
You then actually need to call the function in the perform_update part of the script.